# HG changeset patch
# User Rik <rik@octave.org>
# Date 1450215331 28800
#      Tue Dec 15 13:35:31 2015 -0800
# Node ID c640ea6a1612a107067920db81f261591d47b0d1
# Parent  ebe061d6feeac61cfd95adfefab5982708f6270f
genpropdoc.m: Improve wording for some of the paper* properties.

* genpropdoc.m: Improve wording for some of the paper* properties.

diff --git a/doc/interpreter/genpropdoc.m b/doc/interpreter/genpropdoc.m
--- a/doc/interpreter/genpropdoc.m
+++ b/doc/interpreter/genpropdoc.m
@@ -380,29 +380,28 @@ property.";
       case "numbertitle"
         s.doc = "Display \"Figure\" followed by the numerical figure handle \
 value in the figure title bar.";
 
       case "outerposition"
         s.valid = valid_4elvec;
 
       case "paperorientation"
-        s.doc = "Defines the orientaton of the printed page.  The value for \
-the @code{papersize} property depends upon __prop__.\
-The @code{papersize} values reverse order when __prop__ is switched \
+        s.doc = "The value for the @code{papersize} property depends upon \
+__prop__.  The @code{papersize} values reverse order when __prop__ is switched \
 between @code{\"portrait\"} and either @code{\"landscape\"} or \
 @code{\"rotated\"}.  The value of @code{paperposition} is not dependent upon \
-the value of __prop__, and must be adjusted indepdently when needed."
+the value of __prop__."
 
       case "paperposition"
         s.doc = "Vector @code{[x0 y0 width height]} defining the position and \
 size of the figure (in @code{paperunits} units) on the printed page.  The \
 position @code{[x0 y0]} defines the lower left corner of the figure on the \
-page, and the size is defined by @code{[width height]}.  For output formats, \
-not implicity rendered on paper, the @code{width} and @code{height} define the \
+page, and the size is defined by @code{[width height]}.  For output formats \
+not implicity rendered on paper, @code{width} and @code{height} define the \
 size of the image and the position information is ignored.  __modemsg__.";
         s.valid = valid_4elvec;
 
       case "paperpositionmode"
         s.doc = "If __prop__ is set to @qcode{\"auto\"}, the \
 @code{paperposition} property is automatically computed: the printed \
 figure will have the same size as the on-screen figure and will be centered \
 on the output page.  Setting the __prop__ to @code{\"auto\"} does not modify \
